File: CnnNetLib2/Extensions.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CnnNetLib2
{
    public static class Extensions
    {
        public static double GetDistance(int x1, int y1, int x2, int y2)
        {
            return Math.Sqrt(Math.Pow(x1 - x2, 2) + Math.Pow(y1 - y2, 2));
        }
    }
}

File: Java/src/Extensions.java
/**
 * Created with IntelliJ IDEA.
 * User: smiron86
 * Date: 6/12/13
 * Time: 8:34 PM
 */
public class Extensions {
    public static double GetDistance(int x1, int y1, int x2, int y2)
    {
        return x1 == x2 && y1 == y2
                ? 0
                : Math.sqrt(Math.pow(x1 - x2, 2) + Math.pow(y1 - y2, 2));
    }
}
